### PR TITLE
1. Added style sheet to metrics component as well as made some adjust…

### LINKED ISSUE #53: Full Metric Build

### PR DESCRIPTION
**Pull request by jpatterson933**
This will close #53.

### CODE REVIEW

**Review by StephenODea54 — APPROVED** [review]
🔥🔥🔥🔥🔥

File: client/src/components/Metrics/index.js [hunk 1]
```diff
@@ -11,17 +13,24 @@ export const Metrics = () => {
...
             .catch(error => console.error(error));
     }, []);
+    // filter out tags with 0 posts
+    const tagsWithPosts = tags.filter(tag => tag.posts.length > 0);
+    console.log(tagsWithPosts[0], "test")
 
     return (
-        <>
-            <h1>Metrics</h1>
+        (tags.length > 0)
```
Comment (StephenODea54) [inline client/src/components/Metrics/index.js:21]: If you use the `(tags.length > 0)` for the expression to test in the ternary operator, you no longer need the `tags &&` part in your mapping.

If you wanted to change this up a bit, you could do:

```javascript
return (
        <main id="tags-container">
            {tags && tags.map(tag => {
                return (
                    <div className="tag-card" key={tag.id}>
                        <h1>{tag.name}</h1>
                        {tag.posts.length > 0 ?
                            <p>Total posts with tags {tag.posts.length}</p> :
                            <p>This tag has no posts</p>}
                    </div>
                )
            })}

            {tags.length === 0 ? <h1>No tags exist!</h1> : null}

        </main>
)
```

Comment (jpatterson933) [reply client/src/components/Metrics/index.js:21]: ahhhhh that makes sense. I am going to do a ton more work on this, so I'll leave for now, but I am going to adjust my code.

**Comment by jpatterson933** [comment]
Closing and going to create new branch with new db/schema/oauth changes.

**Pull request closed without merging**